gradle/libs.versions.toml

```diff
@@ -1,14 +1,14 @@
 [versions]
```

Comment:
Some of this third parties libraries are also available in the BOM. Check the catalog definition to see which one: https://github.com/backbase-rnd/mobile-foundation-dependencies/blob/master/catalog/third-parties/gradle/libs.versions.toml